Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1393,6 +1331,18 @@ namespace OpenInfraPlatform {
 				}
 			}
 
+			void SearchExistingVertex(std::stringstream ID, uint32_t index, carve::geom::vector<3> position) const
+			{
+				auto itFound = polygonIndices.find(ID.str());
+				if (itFound != polygonIndices.end()) {
+					index = itFound->second;
+				}
+				else {
+					index = polygon->addVertex(position);

```

Comment:
Does this compile? `polygon` is not defined ...

Reply:
Reseloved: 1b6fe62